File: Learn/2. Big O Notation/OofN.java
public class OofN {
    public static void main(String[] args) {
        Integer[] numbers = { 1, 2, 3, 4, 5 };
        String[] cars = { "Volvo", "BMW", "Ford", "Mazda" };
        callAllNumbers(numbers);
        callAllNumbersAndStrings(numbers, cars);
    }

    static void callAllNumbers(Integer[] numbers) {
        for (int number : numbers)
            System.out.println(number);
    }

    static void callAllNumbersAndStrings(Integer[] numbers, String[] names) {
        for (int number : numbers)
            System.out.println(number);

        for (String name : names)
            System.out.println(name);
    }
}

File: 2. Big O Notation/OofN.js
const calllAllNumbersAndStrings = (numbers, strings) =>{
    for(let i =0; i< numbers.length; i++){
        console.log(numbers[i]);
    }
    for(let i =0; i< strings.length; i++){
        console.log(strings[i]);
    }
}


const callAllNumbers = (numbers) =>{
    for(let i =0; i< numbers.length; i++){
        console.log(numbers[i]);
    }
}

callAllNumbers([1,2,3,4,5]);
calllAllNumbersAndStrings([1,2,3], ['a','b','c']);